discovery-provider/plugins/network-monitoring/src/metrics/index.ts

```diff
@@ -53,7 +53,7 @@ export const generateMetrics = async (run_id: number) => {
 
     const unsyncedUsersCount = await getUnsyncedUsersCount(run_id)
 
-    const ghostUsersCount = await getGhostUsersCount(run_id)
+    const ghostUsersCount = await getUsersWithNullPrimaryClock(run_id)
```

Comment:
why wouldn't you change this everywhere
very confusing currently bc `ghostUsersCount`, `noPrimaryUsersCountGauge`, `getUsersWithNullPrimaryClock` all refer to the same value but have entirely different names

Reply:
oops - I was just blinding renaming symbols in vscode. Should be named a bit better now